kythe/go/extractors/validation/validation.go

```diff
@@ -81,6 +81,8 @@ type Dir struct {
 	Missing int
 }
 
+// accumulate accumulates statistics (currently granular file counts)
```

Comment:
It might also be worth noting that calling this more than once is not idempotent. It's not an exported API, but that could easily happen along multiple paths.

Reply:
Done.